gammapy/irf/psf/map.py

```diff
@@ -523,6 +531,132 @@ def peek(self, figsize=(12, 10)):
         self.exposure_map.reduce_over_axes().plot(ax=axes[2], add_cbar=True)
 
         axes[3].set_title("Containment radius at 1 TeV")
-        self.containment_radius_map(energy_true=2 * u.TeV).plot(
+        kwargs = {self.energy_name:2 * u.TeV}
+        self.containment_radius_map(**kwargs).plot(
             ax=axes[3], add_cbar=True
         )
+
+
+class RecoPSFMap(PSFMap):
+    """Class containing the Map of PSFs in reconstructed energy and allowing to interact with it.
+
+    Parameters
+    ----------
+    psf_map : `~gammapy.maps.Map`
+        the input PSF Map. Should be a Map with 2 non spatial axes.
+        rad and energy axes should be given in this specific order.
+    exposure_map : `~gammapy.maps.Map`
```

Comment:
@LauraOlivera Do we have an exposure map in reconstructed energy here? Computed with the same spectrum as the PSF?

Reply:
I've never seen that needed or used for HAWC... but I guess it is a possibility, and yes it would need to be a reco energy map in that case.